proto/src/determined/run/v1/run.proto

```diff
@@ -0,0 +1,115 @@
+    json_schema: {
+      required: [
+        "id",
+        "is_multitrial",
+        "unmanaged",
+        "forked_from",
+        "external_experiment_id",
+        "resource_pool",
+        "progress",
+        "description",
+        "name",
+        "searcher_type",
+        "searcher_metric"
+      ]
+    }
+  };
+  // The type of searcher for the experiment.
+  string searcher_type = 1;
+  // The searcher metric name for the experiment.
+  string searcher_metric = 2;
+  // Original id of a forked or continued experiment.
+  google.protobuf.Int32Value forked_from = 3;
+  // The id of external experiment
+  string external_experiment_id = 4;
+  // The resource pool the experiment was created in.
+  string resource_pool = 5;
+  // The current progress of a running experiment.
+  float progress = 6;
+  // The description of the experiment.
+  string description = 7;
+  // The experiment name.
+  string name = 8;
+  // Unmanaged experiments are detached.
+  bool unmanaged = 9;
+  // True if the associated experiment is a multitrial experiment
+  bool is_multitrial = 10;
+  // The id of the experiment linked to the run.
+  int32 id = 11;
+}
+
+// Flat run respresentation.
+message FlatRun {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [
+        "id",
+        "start_time",
+        "state",
+        "tags",
+        "checkpoint_size",
+        "checkpoint_count",
+        "project_id",
+        "project_name",
+        "workspace_id",
+        "workspace_name",
+        "parent_archived"
+      ]
+    }
+  };
+  // The id of the run.
+  int32 id = 1;
+  // The time the run was started.
+  google.protobuf.Timestamp start_time = 2;
+  // The time the run ended.
+  google.protobuf.Timestamp end_time = 3;
+  // The current state of the run(trial).
+  determined.trial.v1.State state = 4;
+  // The tags of the associated experiment.
+  // TODO(aaron.amanuel): Create add/remove tags for runs.
+  repeated string labels = 5;
+  // The total size of checkpoints.
+  int64 checkpoint_size = 6;
+  // The count of checkpoints.
+  int32 checkpoint_count = 7;
+  // Signed searcher metrics value.
+  double searcher_metric_value = 8;
+  // The id of external run
+  optional int32 external_run_id = 9;
+  // Trial hyperparameters.
+  string hyperparameters = 10;
+  // summary metrics.
+  optional google.protobuf.Struct summary_metrics = 11;
+  // The id of the user who created the run.
+  optional int32 user_id = 12;
+  // Time in seconds which experiment ran or has been running.
+  optional int32 duration = 13;
+  // The id of the project associated with this experiment.
+  int32 project_id = 14;
+  // The name of the project associated with this experiment.
+  string project_name = 15;
+  // The id of the workspace associated with this experiment.
+  int32 workspace_id = 16;
+  // The name of the workspace associated with this experiment.
+  string workspace_name = 17;
```

Comment:
these comments refer to experiment? Should they refer to run?

Reply:
Updated